lib/activity_model.dart

```diff
@@ -211,41 +211,21 @@ class ActivityModel extends Model {
 
   Color color = Colors.white;
   BlendMode blendMode = BlendMode.dst;
-  String imageId;
-
-  void filterImage(String text) {
-    Color cls;
-    BlendMode blnd;
-    if (text == 'assets/filter_image/default.png') {
-      cls = color;
-      blnd = blendMode;
-    } else if (text == 'assets/filter_image/black_white.png') {
-      cls = Colors.white;
-      blnd = BlendMode.color;
-    } else if (text == 'assets/filter_image/emboss.png') {
-      cls = Colors.red;
-      blnd = BlendMode.colorDodge;
-    } else if (text == 'assets/filter_image/sepia.png') {
-      cls = Colors.red;
-      blnd = BlendMode.darken;
-    } else if (text == 'assets/filter_image/sobel.png') {
-      cls = Colors.deepPurple;
-      blnd = BlendMode.modulate;
-    } else if (text == 'assets/filter_image/vignette.png') {
-      cls = Colors.green;
-      blnd = BlendMode.saturation;
-    } else if (text == 'assets/filter_image/contrast.png') {
-      blnd = BlendMode.darken;
-      cls = Colors.deepOrangeAccent;
-    }
-    print('uuid Image: $imageId');
+  String _imageId;
```

Comment:
Do not call this as _imageId since this could be any selected thing. Call this as _selectedThingId